skype/skype-ps/skype/New-CsCallQueue.md

```diff
@@ -129,6 +129,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OboResourceAccountIds
+The OboResourceAccountIds parameter lets you add resource account with phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts.  This is a list of resource account GUIDs.
+
+Only Call Queue managed by a Teams Channel will be able to use this feature. For more information, please refer to [Manage your support Call Queue in Teams](https://support.microsoft.com/en-us/office/manage-your-support-call-queue-in-teams-9f07dabe-91c6-4a9b-a545-8ffdddd2504e).
+
+```yaml
+Type: List
```

Comment:
```suggestion
Type: Guid
```
It seems that this parameter supports a single GUID as a value:
![image](https://user-images.githubusercontent.com/33589238/121060417-e4bb6000-c7c2-11eb-87cf-5621d880e7e5.png)

Reply:
This parameter accept a single GUID and a list of GUID, so the parameter name has a "s" after it.  Should the type be Guid or List in this case? Please see the following for example:
![image](https://user-images.githubusercontent.com/85360877/121062195-a3eb2900-c779-11eb-94ed-4aa2af447be7.png)
![image](https://user-images.githubusercontent.com/85360877/121062267-b9f8e980-c779-11eb-9c3c-c0233646660c.png)

Reply:
Agree, my mistake. Let's keep List. Thank you!

Reply:
Thanks!  I will resolve this comment and the related ones.